kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -213,6 +212,14 @@
     height: 35px
```

Comment:
It's annoying that GH is not allowing comments on folded lines... :unamused: 

@rayykim 
Anyways, this is a suggestion for line 206, for `:focus` state of the search input field. I really like inset box shadow style for input fields - [check out the one I had on KA Lite login last year](https://github.com/learningequality/ka-lite/pull/3647/files), maybe we can add it here with our own action purple color.

@DXCanas @indirectlylit: The same example contains a style for form fields with errors, whoever is working on it might implement the visual too... :wink: